docs/Concepts/Permissioning/Onchain-Permissioning.md

```diff
@@ -8,6 +8,12 @@ Onchain permissioning uses smart contracts to store and administer the node, acc
 allowlists. Using onchain permissioning enables all nodes to read the allowlists from a single
 source, the blockchain.
 
+!!! important
+
+    Nodes validate account permissions when importing blocks. This means if you disable onchain
+    account permissioning and add transactions to the chain, nodes cannot resync the network if
+    account permissioning is re-enabled.
```

Comment:
I think I'd rephrase this a bit:

When using Onchain Account Permissioning, the node will check permissions when importing blocks. This means that only blocks in which all transactions are from authorized senders will be deemed valid and imported.  If you disable Onchain Account Permissioning and your node accept blocks without enforcing this rule, your node will not be able to resync with other nodes that are enforcing Onchain Account Permissioning rules (your node will be in a forked state).